tests/ci.sh

```diff
@@ -34,11 +34,10 @@ done
 # Dependency mismatcher #
 #########################
 if [ "$MISMATCHER" = "True" ]; then
+
     python3 $(pwd)/utils/dependency_cross_matcher.py --deps_dir $(pwd)
-    mismatcher_result=$?
-    if [ "$mismatcher_result" -ne 0 ]; then
-        error_msg "Found dependencies version mismatches"
-    fi
+
+	python3 $(pwd)/utils/dependency_cross_matcher.py -c --deps_dir $(pwd)/parsec $(pwd)
```

Comment:
nit: spacing?

Reply:
Sorry, what spacing? The added newline?
I thought it looked nicer that way?

Reply:
Oh, I see, the tab. I don't see it in my IDE, and the script is working in the CI. THat's weird

Reply:
Will fix it, it was a "tab" instead of spaces